commands/cli/parse.go

```diff
@@ -357,7 +358,7 @@ func appendStdinAsString(args []string, stdin *os.File) ([]string, *os.File, err
 }
 
 func appendFile(args []files.File, inputs []string, argDef *cmds.Argument, recursive bool) ([]files.File, []string, error) {
-	fpath := inputs[0]
+	fpath := filepath.ToSlash(filepath.Clean(inputs[0]))
```

Comment:
'/' is [invalid](http://stackoverflow.com/questions/10708334/how-can-i-create-files-on-windows-with-embedded-slashes-using-python) in windows filenames. Should be safe. If it were otherwise, `filepath.ToSlash` would have handled the case, I think.